lib/ExpensiMark.js

```diff
@@ -202,6 +202,14 @@ export default class ExpensiMark {
                         return match;
                     }
                     const phoneRegex = new RegExp(`^@${Constants.CONST.REG_EXP.PHONE_PART}$`);
+                    const mention = g2.slice(1);
+                    const metionWithoutSMSDomain = Str.removeSMSDomain(mention);
+                    if (
+                        !Str.isValidMention(match) ||
+                        (new RegExp(`^${Constants.CONST.REG_EXP.PHONE_PART}$`).test(metionWithoutSMSDomain) && !Str.isValidPhoneNumber(metionWithoutSMSDomain))
```

Comment:
can we reuse `phoneRegex` here?

Reply:
@hungvu193 We can't because those are two different Regex

Reply:
ah ok, can you create a variable for it?

Reply:
@hungvu193 please check again